operator_ui/src/actionCreators.ts

```diff
@@ -242,7 +241,7 @@ export const updateBridge = (
 // The calls above will be converted gradually.
 const handleError = (dispatch: Dispatch) => (error: Error) => {
   if (error instanceof jsonapi.AuthenticationError) {
-    sendSignOut(dispatch)
+    dispatch(receiveSignoutSuccess())
```

Comment:
feels strange to say 'successfully signed out' after receiving an auth error?

Reply:
401 means that the cookie is no longer valid, so we need to remove it and unfortunately this is poorly named action that triggers it 😓